crown-court-contribution/src/main/resources/application.yaml

```diff
@@ -60,22 +60,37 @@ spring:
         jwt:
           issuer-uri: ${JWT_ISSUER_URI}
 
+  autoconfigure:
+    exclude: org.springframework.boot.actuate.autoconfigure.tracing.zipkin.ZipkinAutoConfiguration
+
+resilience4j:
+  retry:
+    configs:
+      default:
+        max-attempts: 3
+        wait-duration: 2s
+        enableExponentialBackoff: true
+        exponentialBackoffMultiplier: 2
+        retry-exceptions:
+          - org.springframework.web.reactive.function.client.WebClientRequestException
+          - org.springframework.web.reactive.function.client.WebClientResponseException.BadGateway
+          - org.springframework.web.reactive.function.client.WebClientResponseException.TooManyRequests
+          - org.springframework.web.reactive.function.client.WebClientResponseException.InternalServerError
```

Comment:
Can we make sure this aligns with the changes in the Hardship service as I know we removed the `InternalServerError` in that service?

Reply:
Updated to be in line with Hardship.